lib/boxen/preflight/os.rb

```diff
@@ -1,7 +1,7 @@
 require "boxen/preflight"
 
 class Boxen::Preflight::OS < Boxen::Preflight
-  SUPPORTED_RELEASES = %w(10.8 10.9 10.10 10.11 10.12)
+  SUPPORTED_RELEASES = %w(10.8 10.9 10.10 10.11 10.12 10.13)
```

Comment:
I have been running on High Sierra for some time now and see no problems related to the OS itself. Just FYI

Reply:
@webflo Are you able to update this line to just be `10.13` per #221? I'll merge this following that.